client/zh_cn.editable.json

```
@@ -284,34 +284,34 @@
     "projectelo.button.hidden": "隐藏",
     "projectelo.button.toggle": "切换",
     "projectelo.button.profile": "资料",
-    "projectelo.button.filter": "过滤器",
-    "projectelo.button.filter_archived_only": "仅显示封存的回放",
-    "projectelo.button.archive_replay": "封存回放",
-    "projectelo.button.unarchive_replay": "解封回放",
+    "projectelo.button.filter": "筛选",
+    "projectelo.button.filter_archived_only": "仅显示打包的回放",
+    "projectelo.button.archive_replay": "打包回放",
+    "projectelo.button.unarchive_replay": "解压回放",
     "projectelo.button.watch_replay": "观看回放",
     "projectelo.button.settings": "设置",
     "projectelo.button.reset_world": "重置世界",
     "projectelo.button.recent_matches": "最近比赛",
     "projectelo.button.link_third_party": "绑定 %s",
     "projectelo.button.unlink_third_party": "解绑 %s",
     "projectelo.button.public_third_party": "公开",
-    "projectelo.button.public_third_party.description": "决定是否要在你的个人资料上显示你的 %s 标签/名字。",
+    "projectelo.button.public_third_party.description": "是否要在您的个人资料上显示你的 %s 标签/名字。",
     "projectelo.button.leaderboard": "排行榜",
     "projectelo.button.undo": "撤销",
     "projectelo.button.redo": "重做",
     "projectelo.button.download": "下载",
     "projectelo.button.downloading": "下载中...",
-    "projectelo.button.close_instance": "关闭这个 Minecraft 实例",
+    "projectelo.button.close_instance": "关闭当前 Minecraft 实例",
     "projectelo.button.patreon_support": "Patreon 赞助",
     "projectelo.button.seed_change_vote": "更换种子",
     "projectelo.button.draw_vote": "发起平局投票",
     "projectelo.button.report": "举报",
-    "projectelo.button.report_toxic_chat": "嘴臭",
-    "projectelo.button.report_hacking": "开挂",
+    "projectelo.button.report_toxic_chat": "人身攻击",
```

```suggestion
    "projectelo.button.report_toxic_chat": "发言不当",
```

zh-cn: toxic chat 并不完全指人身攻击。